org.kde.CrowTranslate.yaml

```diff
@@ -0,0 +1,64 @@
+id: org.kde.CrowTranslate
+sdk: org.kde.Sdk
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # The app supports capturing screen via KWin and Mutter DBus APIs, but
+  # access to it seems overly broad in spite of the app also supporting XDP API
+  #- --talk-name=org.kde.KWin
+  #- --talk-name=org.gnome.Shell
```

Comment:
This should go into a README for the repo to make it easier to find.

Maybe also in a message in the application when running as Flatpak.

Reply:
Removed completely. `org.kde.KWin` and `org.gnome.Shell` aren't necessary.